ocaml/lambda/debuginfo.ml

```diff
@@ -247,8 +277,14 @@ let compare { dbg = dbg1; } { dbg = dbg2; } =
   in
   loop (List.rev dbg1) (List.rev dbg2)
 
-let hash { dbg; } =
-  List.fold_left (fun hash item -> Hashtbl.hash (hash, item)) 0 dbg
+let compare { dbg = dbg1; assume_zero_alloc = a1; }
+      { dbg = dbg2; assume_zero_alloc = a2; } =
```

Comment:
I think this may cause a loss of sharing
in `Emitaux`. If it is to be fixed, the
change should probably be in `Emitaux`.

Reply:
You are right, in `Emitaux` we should ignore the `assume` field.